client/my-sites/stats/activity-log-banner/progress-banner.jsx

```diff
@@ -33,26 +41,28 @@ function ProgressBanner( {
 			<p>
 				{ translate(
 					"We're in the process of restoring your site back to %s. " +
-						"{{br/}}You'll be notified once it's complete.",
-					{
-						args: applySiteOffset( moment.utc( timestamp ) ).format( 'LLLL' ),
-						components: {
-							br: <br />,
-						},
-					}
+						"You'll be notified once it's complete.",
+					{ args: applySiteOffset( moment.utc( timestamp ) ).format( 'LLLL' ) }
 				) }
 			</p>
 
 			<div>
-				<em>{ translate( "We're on it! Your site is being restored." ) }</em>
+				<em>{ restoreStatusDescription }</em>
+				<ProgressBar
+					className={ status === 'queued' ? 'activity-log-banner__progress-bar--queued' : null }
+					isPulsing
+					value={ status === 'queued' ? 100 : percent }
```

Comment:
do we want to show `100` if it's `queued`? is that the end of `running`?

Reply:
@MichaelArestad should we even show the progress bar if the site is waiting for a restore queue slot?